src/content/ueber-uns/abteilungen.svx

```diff
@@ -1,57 +1,49 @@
 
-Die Infanterie setzt sich aus Schützen, MG-Schützen, AT-Schützen, Präzisionsschützen zusammen. Ihre Bewegung und der Feuerkampf erfolgen ausschließlich zu Fuß. Alternativ können sich die Soldaten auf Fahrzeugen aufgesessen bewegen, auch motorisierte Infanterie genannt. Die Infanterie kann vielseitig eingesetzt werden. Sie erweist sich besonders im urbanen Gebiet und unwegsamen Gelände als effektiv, da Fahrzeuge dort nur eingeschränkt beweglich sind.
+_Bemerkung_: Infanterie, die aufgesessen auf [geschützten Fahrzeugen (DEAD)](../fuhrpark/geschuetzte-fahrzeuge) agiert, nennt man **motorisierte Infanterie**.
 
-### Sanitätsdienst
-
-Der Sanitätsdienst setzt sich aus Sanitätern und Ärzten zusammen. Er wird von ptolemy organisiert und verwaltet. Sanitäter sind in Trupps und Gruppen der Infanterie vertreten und übernehmen die fortgeschrittene Versorgung von verwundeten Soldaten im Feld. Ärzte sind vorwiegend bei Spezialanwendungen wie dem MEDEVAC oder dem BAT anzutreffen.
-
-### Aufklärung
-
-Die Aufklärung setzt sich aus Scharfschützen, Spottern und JTACs zusammen. Scharfschützen und Spotter sind meist im Trupp oder Buddy-Team zur Aufklärung oder Bekämpfung feindlicher Kräfte auf große Distanzen anzutreffen. JTACs sind entweder im Aufklärungstrupp oder in Infanteriegruppen vertreten. Ihre Hauptaufgabe ist die Koordinierung vom CAS auf feindliche Ziele.
+### Panzertruppen
 
-## Panzertruppen
+Die Panzertruppen sind ein Truppengattungsverbund aus drei Truppengattungen: Die Panzertruppe, die Panzergrenadiertruppe und die Artillerietruppe. Die Truppengattungen dieses Verbundes werden von Kevin organisiert und verwaltet.
 
-Die Abteilung der Streitkräfte wird von Kevin organisiert und verwaltet. Sie umfasst insgesamt drei Truppengattungen: Die mechanisierte Infanterie, die Kampfpanzer und die Artillerie. Alle Truppengattungen der Panzertruppen umfassen drei Rollen: Den Kommandanten, den Richtschützen und den Fahrer.
+Die Truppengattung der **_Panzertruppe_** setzt sich aus Panzerbesatzungen zusammen, die [Kampfpanzer (DEAD)](../fuhrpark/kampfpanzer) bedienen. Die Truppengattung der **_Panzergrenadiertruppe_** setzt sich aus Panzerbesatzungen und Infanteristen (hier: _Panzergrenadieren_) zusammen, die gemeinsam auf einem [Schützenpanzer (DEAD)](../fuhrpark/schuetzenpanzer) agieren. Die Truppengattung der **_Artillerietruppe_** setzt sich aus Panzerbesatzungen zusammen, die [Artilleriepanzer (DEAD)](../fuhrpark/artilleriepanzer) und [Raketenwerfer-Artilleriesysteme (DEAD)](../fuhrpark/raketenwerferartillerie) bedienen. Eine Panzerbesatzung besteht aus einem [Kommandanten](../panzertruppen/kommandant), einem [Richtschützen](../panzertruppen/richtschuetze) und einem [Fahrer](../panzertruppen/fahrer).
```

Comment:
I'm noticing some repeating sentence structure, maybe consider using a table to convey the information?

Reply:
Not quite sure how. It's quite a bit of text for a table.

Reply:
We can keep that in the back of our heads for later.